src/index.html

```diff
@@ -5,8 +5,338 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a href="#home" class="header__logo-link">
+          <img src="./images/logo.png" alt="Logo of the company">
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#smartphones" class="nav__link">Smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#laptopsAndComputers"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#gadgets" class="nav__link">Gadgets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#tablets" class="nav__link">Tablets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#photo" class="nav__link">Photo</a>
+            </li>
+            <li class="nav__item">
+              <a href="#video" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="catalog">
+      <div class="catalog__container">
+        <div class="card card--bottom card--indent-right" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="Apple A1419 iMac monoblock 5k"
+          >
+
+          <h3 class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+
+          <ul class="stars stars--4 stars--after-element">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
```

Comment:
`Reviews: 5` should be as html element not pseudo. Also stars should work with modifiers (1-5).